examples/tracer-web/index.js

```diff
@@ -1,11 +1,83 @@
 import { ConsoleSpanExporter, SimpleSpanProcessor } from '@opentelemetry/tracing';
 import { WebTracer } from '@opentelemetry/web';
 import { DocumentLoad } from '@opentelemetry/plugin-document-load';
+import { ZoneScopeManager } from '@opentelemetry/scope-zone';
 
 const webTracer = new WebTracer({
   plugins: [
     new DocumentLoad()
   ]
 });
-
 webTracer.addSpanProcessor(new SimpleSpanProcessor(new ConsoleSpanExporter()));
+
+const webTracerWithZone = new WebTracer({
+  scopeManager: new ZoneScopeManager(),
+  plugins: [
+    new DocumentLoad()
+  ]
+});
+webTracerWithZone.addSpanProcessor(new SimpleSpanProcessor(new ConsoleSpanExporter()));
+
+console.log('Current span is window', webTracerWithZone.getCurrentSpan() === window);
+
+// example of keeping track of scope between async operations
+const prepareClickEvent = () => {
+  const url1 = 'https://raw.githubusercontent.com/open-telemetry/opentelemetry-js/master/package.json';
+  const url2 = 'https://raw.githubusercontent.com/open-telemetry/opentelemetry-js/master/packages/opentelemetry-web/package.json';
+
+  const element = document.getElementById('button1');
+  let mainSpan = webTracerWithZone.startSpan('main-span');
+  webTracerWithZone.bind(element, mainSpan);
```

Comment:
In this scenario, is `main-span` going to last for the full user session on the page?

My sense is that the page session should be related by an attribute on all spans (say linking all interaction spans back to the document load root span), but that having an actual span for the user's whole session in my mind isn't super useful.

I realize this is just an example, but I think it informs how people will use it. Would it make more sense to start a new span each time the button is clicked such that it has no parent (i.e. its a root span)? In that case, can we remove the `bind` here?

Reply:
`bind` method provides the same functionality like the `AsyncHooksScopeManager`. It binds the provided span to the element / target. This is example how to use bind. The case you mentioned about is the case for `with`. Those are 2 different functionality and it's up to user to decide which one to choose for which case. But here the user can find examples of using them both.

Reply:
here https://github.com/open-telemetry/opentelemetry-js/blob/b1255217a6f491756335fcb0945f4c8f205e0200/packages/opentelemetry-scope-base/src/types.ts#L38  and here https://github.com/open-telemetry/opentelemetry-js/blob/b1255217a6f491756335fcb0945f4c8f205e0200/packages/opentelemetry-scope-base/src/types.ts#L38